app/api/auth.py

```diff
@@ -138,6 +140,24 @@ def login_user(provider, auth_code):
             'client_id': provider_class.get_client_id(),
             'client_secret': provider_class.get_client_secret()
         }
+        if not payload['client_id'] and payload['client_secret']:
```

Comment:
Wrong condition

Reply:
Yes, this is wrong. I was just waiting for the #5589 to get merged so that I can import the helper function implemented there.

Reply:
NotImplementedError